tests/auth/registration/registretion.spec.mjs

```diff
@@ -0,0 +1,236 @@
+        const emailInput = popup.locator('input#signupEmail')
+        const passwordInput = popup.locator('input#signupPassword')
+        const repeatPasswordInput = popup.locator('input#signupRepeatPassword')
+
+        await nameInput.fill(name)
+        await lastNameInput.fill(lastName)
+        await emailInput.fill(email)
+        await passwordInput.fill(password)
+        await repeatPasswordInput.fill(repeatPassword)
+        const passwordErrorMessage = popup.locator('p', {hasText: 'Password has to be from 8 to 15 characters long and contain at least one integer, one capital, and one small letter'})
+
+        await expect(passwordErrorMessage, 'Error message should be shown when wrong data entered in Password field').toBeVisible()
+        await expect(passwordErrorMessage, 'Error message should be shown when wrong data entered in Password field').toHaveText('Password has to be from 8 to 15 characters long and contain at least one integer, one capital, and one small letter')
+        await expect(passwordErrorMessage, 'Password input should has red border when wrong data entered in Password field').toHaveCSS('border-color', 'rgb(220, 53, 69)')
+
+    })
+
+    test('Re-entered password do not match with Password', async ({page})=>{
+
+        await page.goto('/')
+
+        const name = 'Sasha'
+        const lastName = 'Dzyu'
+        const email = 'aqa-dzyu@mail.com'
+        const password = '1234Password'
+        const repeatPassword = '12333Password'
+
+        const signUpButton = page.locator('button.hero-descriptor_btn.btn.btn-primary')
+        await expect(signUpButton, 'Sign up button should be visible').toBeVisible()
+        await expect(signUpButton, 'Sign up button should be enabled').toBeEnabled()
+        await signUpButton.click()
+
+        const popup = page.locator('div.modal-content')
+        await expect(popup, "Sign up popup should be visible").toBeVisible()
+
+        const nameInput = popup.locator('input#signupName')
+        const lastNameInput = popup.locator('input#signupLastName')
+        const emailInput = popup.locator('input#signupEmail')
+        const passwordInput = popup.locator('input#signupPassword')
+        const repeatPasswordInput = popup.locator('input#signupRepeatPassword')
+
+        await nameInput.fill(name)
+        await lastNameInput.fill(lastName)
+        await emailInput.fill(email)
+        await passwordInput.fill(password)
+        await repeatPasswordInput.fill(repeatPassword)
+
+        await nameInput.click()
+
+        const RepeatPasswordErrorMessage = popup.locator('p', {hasText: 'Passwords do not match'})
+
+        await expect(RepeatPasswordErrorMessage, 'Error message should be shown when wrong data entered in Re-enter password field').toBeVisible()
+        await expect(RepeatPasswordErrorMessage, 'Error message should be shown when wrong data entered in Re-enter password field').toHaveText('Passwords do not match')
```

Comment:
you search by text and then make assertion for the same text, it is a bit strange. It always will be true

Reply:
same for other tests

Reply:
use class selectors